README.md

```diff
@@ -26,7 +39,7 @@ The JFrog extension automatically triggers a scan of the project's npm dependenc
 To invoke a scan manually, click on the Refresh ![Refresh](resources/readme/refresh.png) button or click on *Start Xray Scan* from within the **package.json** (above the *dependencies* section):
 ![Refresh](resources/readme/gifs/refresh.gif)
 
-View existing issues
+View existing issues:
 ![Vulnerabilities](resources/readme/gifs/show_vulnerabilities.gif)
 
 View licenses directly from within the **package.json**:
```

Comment:
**package.json**:
 -->
**package.json**, **requirements.txt** or **go.mod**.

(I noticed at least one more place where **requirements.txt** and **go.mod** should be added).